home.html

```diff
@@ -0,0 +1,171 @@
+          <p class="home-page-text-p">Join us into the biggest congrest about block-chain technology
+            and know the new developments and improvements with companies
+            that are already developing applications that will renew the world
+          </p>
+        </div>
+        
+        <!--    where and when descrip    -->
+        
+        <div class="home-page-info">
+          <h2 class="home-page-info-date">2023.10.15 ~ 16</h2>
+          <h3 class="home-page-info-place">see you on: Dubai Exhibition Centre</h3>
+        </div>
+      </div>
+    </section>
+
+      <!--    schedule main program    -->
+
+      <div class="home-page-main-program">        
+        <div class="home-page-main-program-title-content">
+          <h2 class="home-page-main-program-title">Main Program</h2>
+          <div class="home-page-main-program-border"></div>
+        </div>
+        <div class="home-page-main-program-container">
+          <div class="home-page-main-program-1">
+            <img src="image/program_icon_01.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-1-oratory">Lecture</h2>
+            <p class="home-page-main-program-paragraph"> Listen to the speakers from various countries about the messages of sharing and opening.</p>
+          </div>
+          <div class="home-page-main-program-2">
+            <img src="image/program_icon_02.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-2-oratory">Exhibition</h2>
+            <p class="home-page-main-program-paragraph"> Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+          </div>
+          <div class="home-page-main-program-3">
+            <img src="image/program_icon_03.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-3-oratory">Forum</h2>
+            <p class="home-page-main-program-paragraph"> Have the time to share your thoughts and opinions with experts for each topic.</p>
+          </div>
+          <div class="home-page-main-program-4">
+            <img src="image/program_icon_04.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-4-oratory">Workshop</h2>
+            <p class="home-page-main-program-paragraph"> Try creating your own work using open source license rather than just watching at it.</p>
+          </div>
+          <div class="home-page-main-program-5">
+            <img src="image/program_icon_05.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-5-oratory">Ignite</h2>
+            <p class="home-page-main-program-paragraph"> get opportunities to network with CC affiliates around the world, also after the summit.</p>
+          </div>
+        </div>
+
+          <!--    button to submit    -->
+
+        <button class="home-page-info-joinNowBtn">GET YOUR PASS</button>
```

Comment:
kindly consider removing the borders on this button and just underline it to me the design requirements